deps/libgit2.mk

```diff
@@ -104,8 +104,9 @@ $(LIBGIT2_SRC_PATH)/libgit2-remote-push-NULL.patch-applied: $(LIBGIT2_SRC_PATH)/
 	echo 1 > $@
 
 $(build_datarootdir)/julia/cert.pem: $(CERTFILE)
+	$(RM) $@
 	mkdir -p $(build_datarootdir)/julia
-	-cp $(CERTFILE) $@
+	cp $(CERTFILE) $@
```

Comment:
The posix spec says that the `cp -f` will:
> If a file descriptor for a destination file cannot be obtained, as described in step 3.a.ii., attempt to unlink the destination file and proceed.

From what I can tell, unlink should act regardless of file permissions (i.e. depend only on the directory permissions), so `cp -f` should be okay.